modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -26,57 +26,44 @@ The following table contains examples of the URLs you need to supply, depending
 | Token Introspection | {BASE URL}/introspect | {BASE URL}/introspect | {BASE URL}/introspect
 |===
 
-. Log into the master organization in Anypoint Platform as Administrator.
-. In Anypoint Platform, click *Access Management > External Identity*.
+. Sign into the master organization in Anypoint Platform with the *Organization Administrator* role.
+. In Anypoint Platform, click *Access Management*. 
+. In the menu on the left, click *External Identity*.
+. In the drop-down, select *Client Management > OpenID Connect Dynamic Client Registration*.
 +
 image::select-openid-client-mgmt.png[height=310,width=452]
-. From *Client Management*, select *OpenID Connect Dynamic Client Registration*.
-+
-The External Identity - Client Management OpenID Connect Dynamic Client Registration form appears.
-+
-. Fill in the following required fields after obtaining values from your identity provider’s configuration:
-+
-* Client Registration URL
-+
-The URL to dynamically register client applications as a client application for your identity provider.
-+
-* Authorize Header
-+
-The authorization header for dynamic client registration requests. This is an optional field under the *Advanced Settings* link. This header is required if the provider restricts registration requests to authorized clients.
-+
-** Okta: This value is `SSWS ${api_token}`, where `api_token` is an API token created through Okta.
-+
-** ForgeRock: This value is `Bearer ${api_token}`, where `api_token` is an API token created through ForgeRock.
-+
-** Salesforce: This value is `Bearer ${api_token}`, where `api_token` is an API token created through Salesforce.
-+
-* Token Introspection Client: Client ID &amp; Client Secret
-+
-The client ID and client secret for an existing client in your IdP capable of introspection of _all_ tokens from all clients.
-+
-** Okta: This value should be a "Confidential" client.
-+
-** ForgeRock: This value should be a "Confidential" client.
-+
-** Salesforce: This value should be a "Confidential" client.
-+
-* Authorize URL
-+
-The URL where the user authenticates and grants OpenID Connect client applications access to the user's identity.
-+
-* Token URL
-+
-The URL that provides the user’s identity, encoded in a secure JSON Web Token.
-+
-. Save your configuration.
 +
+The *External Identity* page appears.
++
+. In *Client Management*, after obtaining values from your identity provider’s configuration, fill in the following required fields in each section:
++
+* *Dynamic Client Registration*
+ ** *Issuer*: URL that the OpenID provider asserts is its trusted issuer.
+ ** *Client Registration URL*: The URL to dynamically register client applications as a client application for your identity provider. +
+  *** For Okta, this value is `SSWS ${api_token}`, where `api_token` is an API token created through Okta.
```

Comment:
These bullet points should be under the Authorization Header field. Not the client registration url.

Reply:
Moved.